configargparse.py

```diff
@@ -1104,23 +1104,6 @@ def parse_known_args(
         self.write_config_file(namespace, output_file_paths, exit_after=True)
         return namespace, unknown_args
 
-    def get_source_to_settings_dict(self):
```

Comment:
This method has been part of the configargparse API for a long time. Why delete it? 
How do you know that no other project relies on it?